_api-reference/index-apis/flush.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: default
+title: Flush
+parent: Index APIs
+nav_order: 36
+---
+
+# Flush
+
+**Introduced 1.0**
+{: .label .label-purple }
+
+The Flush API takes all in-memory operations and stores them to segments on disk. Operations flushed to an index segment are no longer needed in transaction logs in the event of a cluster restart because these operations are now stored in the Lucene Index. 
```

Comment:
```suggestion
The Flush API takes all in-memory operations and stores them to segments on disk. Operations flushed to an index segment are no longer needed in transaction logs during a cluster restart because these operations are now stored in the Lucene Index. 
```